presets/4.4/tune/karate/karate_brushless_whoop.txt

```diff
@@ -0,0 +1,162 @@
+#$ TITLE: Karate brushless whoop
+#$ FIRMWARE_VERSION: 4.4
+#$ CATEGORY: TUNE
+#$ STATUS: EXPERIMENTAL
+#$ KEYWORDS: karate, 2s, race, brushless,  whoop 5 inch, 5", sugarK, KarateBrot
```

Comment:
i guess need to remove 5" from keywords, and add 65mm or whatever size whoops are?

Reply:
dam